README.md

```diff
@@ -40,20 +40,19 @@ We offer a choice of layernorm, scalenorm and RMSNorm easily configured by chang
 
 - **Zero Redundancy Optimizer (ZeRO):** 
 
-    - ZeRO stage 1 works seamlessly with NeoX, while ZeRO stage 2 requires pipeline parallelism be set to 0. We are additionally working on integrating ZeRO 3 into the codebase.
+    - ZeRO stage 1 works seamlessly with NeoX, while ZeRO stage 2 requires pipeline parallelism be set to 0, as ZeRO performs pipeline parallelism internally. We are additionally working on integrating ZeRO 3 into the codebase.
```

Comment:
"ZeRO performs pipeline parallelism internally" This is inaccurate

Reply:
Fixed